extras/free-books.md

```diff
@@ -48,6 +48,9 @@ Name | Author(s)
 
 ### Math (Discrete Math)
 
+:-- | :--:
+[Applied Discrete Structures](http://faculty.uml.edu/klevasseur/asd2/) | Alan Doerr, Ken Levasseur
```

Comment:
Should be http://faculty.uml.edu/klevasseur/ads2/ instead of http://faculty.uml.edu/klevasseur/asd2/